src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Constants.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.Sbom.Extensions.Entities;
+
+namespace Microsoft.Sbom.Parsers.Spdx30SbomParser;
+
+internal static class Constants
+{
+    internal const string SPDXName = "SPDX";
+    internal const string SPDXVersion = "3.0";
+    internal const string DataLicenceValue = "CC0-1.0";
```

Comment:
It looks like the following are defined but not used--not sure what that means, though...
- `DataLicenceValue`
- `SPDXDocumentIdValue`
- `RootPackageIdValue`
- `DataLicenseHeaderName`
- `SPDXDocumentNameFormatString`

Reply:
Everything except DataLicenseHeaderName is used in generator.cs. I will remove DataLicenseHeaderName from Constants.cs